docs/modules/publisher-directmethods.md

```diff
@@ -65,46 +61,48 @@ The `_V1` direct methods  uses the  payload schema as described below:
+| `DataSetWriterGroup`                | no        | String  | EndpointUrl               | The data set writer group collecting datasets defined for a certain <br>endpoint uniquely identified by the above attributes. <br>This attribute is used to identify the session opened into the <br>server. The default value consists of the EndpointUrl string, <br>followed by a deterministic hash composed of the <br>EndpointUrl, UseSecurity, OpcAuthenticationMode, UserName and Password attributes.                                                                                                                                                                                                          |
+| `DataSetWriterId`                   | no        | String  | DataSetPublishingInterval | The unique identifier for a data set writer used to collect <br>OPC UA nodes to be semantically grouped and published with <br>the same publishing interval. <br>When not specified a string representing the common <br>publishing interval of the nodes in the data set collection. <br>This attribute uniquely identifies a data set <br>within a DataSetWriterGroup. The uniqueness is determined <br>using the provided DataSetWriterId and the publishing <br>interval of the grouped OpcNodes.  An individual <br>subscription is created for each DataSetWriterId. |
+| `DataSetPublishingInterval`         | no        | Integer | null                      | The publishing interval used for a grouped set of nodes <br>under a certain DataSetWriter. When defined it <br>overrides the OpcPublishingInterval value in the OpcNodes <br>if grouped underneath a DataSetWriter. <br>_Note:_ the DataSetPublishingInterval is overwritten if an<br> OpcPublishingInterval is set at the node level                                                                                                                                                                                                                                  |
+| `DataSetPublishingIntervalTimespan` | no        | String | null                      | The publishing interval used for a grouped set of nodes <br> expressed as a Timespan string ({d.hh:mm:dd.fff}).<br>When both Intervals are specified, the <br> Timespan will win and be used for the configuration. <br>_Note:_ the DataSetPublishingInterval is overwritten if an<br> OpcPublishingInterval is set at the node level                                                                                                                                                                                                                                  |
```

Comment:
>grouped set of nodes [](http://example.com/codeflow?start=124&length=20)

This is hard to understand for the user. Is not the DataSet itself a group of nodes identified by the same publishing interval?

Reply:
Changed the wording. @hansgschossmann, please have another look at it.